src/PerfCounters.cc

```diff
@@ -575,7 +599,21 @@ bool PerfCounters::is_ticks_attr(const perf_event_attr& attr) {
   perf_event_attr tmp_attr = attr;
   tmp_attr.sample_period = 0;
   tmp_attr.config &= ~IN_TXCP;
-  return memcmp(&ticks_attr, &tmp_attr, sizeof(attr)) == 0;
+  if (memcmp(&ticks_attr, &tmp_attr, sizeof(attr)) == 0) {
+    return true;
+  }
+  return false;
+}
```

Comment:
Why did you change this?

Reply:
Thanks for catching this. I'd simply forgotten to revert to the old code after briefly returning true for ticks_attr and minus_ticks_attr.